src/style.css

```diff
@@ -0,0 +1,78 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search_form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.search_form:before {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: 100%;
+  content: "";
+  height: 19px;
+  width: 19px;
+  left: 26px;
+  top: 50%;
+  transform: translate(0, -50%);
+  position: absolute;
+}
+
+.search_form--sm:before {
+  height: 11px;
+  left: 13px;
+  width: 11px;
+}
+
+.search_field {
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+  display: block;
+  font-family: "Avenir", sans-serif;
```

Comment:
move `font-family: "Avenir", sans-serif;` to `html` tag and use `font-family: inherit;` here